tests/integration/test_evolve.py

```diff
@@ -56,18 +56,27 @@ def evolve_all(self, t, skip=None):
         for H_name in hamiltonians.get_names():
             if H_name in skip:
                 continue
-
+            if H_name == 'syk' and 'slow' in self.skip_flags:
+                continue
             with self.subTest(H = H_name):
                 H = getattr(hamiltonians, H_name)()
                 self.evolve_check(H, t)
 
     def test_zero(self):
-        self.evolve_all(0.0)
+        if config.L < 20:
```

Comment:
maybe we switch these conditionals to another flag instead of hardcoding spin chain size